docs/DeveloperGuide.md

```diff
@@ -367,11 +369,12 @@ The following activity diagrams summarise what happens when a user executes the
 The following sequence diagram summarises what happens when the user inputs an `show` command together with a student specified by name.
 
 ![ShowSequenceDiagram](images/ShowSequenceDiagram.png)
+![ShowStatsFromModelSequenceDiagram](images/ShowStatsFromModelSequenceDiagram.png)
 
 <br>
 
 
-### Alias feature
+### Alias : `alias`
```

Comment:
Do you think the non-code portions of these should be a little longer? 
"Alias : `alias`" feels really weird and repetitive. Maybe keeping the feature would help. i.e. "Alias feature : `alias`".

Or perhaps a short description of the command that feels less awkward? e.g. "Adding students : `addstudent`" instead of "Add student : `addstudent`"

Reply:
Okay then how about this?

- Importing
- Adding student feature
- Adding group feature
- Adding allocation feature
- Adding score feature
- Searching feature
- Showing analysis
- Adding alias feature

I removed the command keywords because they are already shown the the subtitles (e.g. `How the `addstudent` command works)

Reply:
I think you can drop the "feature".
`<verb>ing <plural noun>` is probably a reliable convention to stick to. Maybe:

* Importing data
* Adding students
* Adding groups
* Adding students to groups
* Adding scores
* Searching for students
* Showing analysis
* Adding aliases

Reply:
Okay sure.

Do you think it's better to reorganise the features to be the same order as the UG (i.e. putting import section between search and show)?

And should the titles of the UG be renamed to the above form `<verb>ing <plural noun>` too?

UG ToC currently looks like this
![image](https://user-images.githubusercontent.com/63232300/140637605-1eec092c-0224-4786-bb11-6b9269e3ffed.png)

Reply:
Oh I think the UG headers look great. 
"Adding a student" is more accurate than "Adding students".

Following the UG sounds like a great idea too. Thanks for pointing this out!